src/DeprecatedClassFacade.php

```diff
@@ -67,7 +67,7 @@ public function __call( $name, $arguments ) {
 	 */
 	public static function __callStatic( $name, $arguments ) {
 		_deprecated_function(
-			static::$facade_over_classname . '::' . $name,
+			static::class . '::' . $name,
```

Comment:
What is the difference between `get_class( $this )` and `static::class`?

Reply:
There won't be an instance (`$this`) in static calls.

Reply:
Right, of course. 🤦‍♂️ Can `static::class` be used in line 50 instead of `get_class( $this )` though?

Reply:
Yep! Looks like it can, commit incoming.

Reply:
Fixed in 0202fe8974c0bcf47cccde3f1116c70883b7d341.